post-processor/amazon-import/post-processor.go

```diff
@@ -63,6 +64,10 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		p.config.S3Key = "packer-import-{{timestamp}}.ova"
 	}
 
+	if p.config.LicenseType == "" {
+		p.config.LicenseType = "BYOL"
```

Comment:
Of course, if you light up a Windows box and you use the default AWS licensing, I hope you remembered to pack your wallet. 😁